src/Invoice_GenUI/Invoice_GenUI/Models/HttpServices/BaseService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Invoice_GenUI.Models.Services
+{
+    public class BaseService
+    {
+        public HttpClient CreateHttpClient()
```

Comment:
:seedling: for another PR - the HttpClient can be set up in App.xaml.cs and dependency injected in.

One of the few optimizations I can teach you about before the end of the project, if there's time.